drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTableViewImpl.java

```diff
@@ -0,0 +1,640 @@
+                pop.hide();
+            }
+
+        } );
+        DirtyableHorizontalPane horiz = new DirtyableHorizontalPane();
+        horiz.add( box );
+        horiz.add( addButton );
+
+        pop.addAttribute( new StringBuilder( GuidedDecisionTableConstants.INSTANCE.Metadata1() )
+                                  .append( GuidedDecisionTableConstants.COLON ).toString(), horiz );
+        pop.addAttribute( GuidedDecisionTableConstants.INSTANCE.Attribute(),
+                          list );
+        pop.show();
+    }
+
+    @Override
+    public void newExtendedEntryConditionColumn() {
+        final ConditionCol52 column = new ConditionCol52();
+        final ConditionPopup popup = new ConditionPopup( model,
+                                                         oracle,
+                                                         presenter,
+                                                         new ConditionColumnCommand() {
+                                                             public void execute( final Pattern52 pattern,
+                                                                                  final ConditionCol52 column ) {
+                                                                 presenter.appendColumn( pattern,
+                                                                                         column );
+                                                             }
+                                                         },
+                                                         column,
+                                                         true,
+                                                         !access.isEditable() );
+        popup.show();
+    }
+
+    @Override
+    public void newLimitedEntryConditionColumn() {
+        final ConditionCol52 column = new LimitedEntryConditionCol52();
+        final ConditionPopup popup = new ConditionPopup( model,
+                                                         oracle,
+                                                         presenter,
+                                                         new ConditionColumnCommand() {
+                                                             public void execute( final Pattern52 pattern,
+                                                                                  final ConditionCol52 column ) {
+                                                                 presenter.appendColumn( pattern,
+                                                                                         column );
+                                                             }
+                                                         },
+                                                         column,
+                                                         true,
+                                                         !access.isEditable() );
+        popup.show();
+    }
```

Comment:
The two methods above are the same except for the first line...perhaps reuse the code instead?